### PR TITLE
[TSK-243] add toggle for sending email

### PR DESCRIPTION
**Pull request by Hoocki**

### CODE REVIEW

File: notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import com.qpay.libs.models.PaymentNotification;
+import com.qpay.notificationmanager.service.NotificationService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
```
Comment (adsukharev) [inline notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java]: you don't need it

File: notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@ConditionalOnProperty(name = "email.enabled", havingValue = "false")
+public class MockNotificationService implements NotificationService {
+
+    public void sendMessage(final PaymentNotification paymentNotification) {
+        log.info("Mock sending message to email: {}", paymentNotification.emailTo());
```
Comment (adsukharev) [inline notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java]:
```suggestion
        log.info("Mock email message sent to email: {}", paymentNotification.emailTo());
```

**Review by adsukharev — APPROVED** [review]

File: notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java (hunk 1 above)
Comment (Hoocki) [reply notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java]: but without it i couldn't initialize service

Comment (adsukharev) [reply notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java]: I mean, you could use no args constructor to show you don't have any fields. but you can leave this one

Comment (Hoocki) [reply notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/MockNotificationService.java]: ok

**Pull request merged**